apps/docs/src/app/core/component-docs/split-button/split-button-docs.component.html

```diff
@@ -11,7 +11,8 @@
     <code class="code-snippet">[fdType]="'negative'"</code>
     <code class="code-snippet">[fdType]="'attention'"</code>
     <code class="code-snippet">[fdType]="'transparent'"</code>. Leave
-    <code class="code-snippet">[fdType]=""</code> empty to use Action button
+    <code class="code-snippet">[fdType]=""</code> empty to use Action button<br />
+    To add a custom title use the <code class="code-snippet">mainActionTitle</code> input directive.
```

Comment:
"Input directive" is confusing, it's just an input

Reply:
changed